app/assets/javascripts/surveys.js

```diff
@@ -175,6 +175,111 @@
           title: 'Tell us about your visit to GOV.UK',
           surveyCta: 'Answer 2 quick questions to help us make GOV.UK better',
         }
+      },
+      {
+        identifier: 'scrutinise_government',
+        surveyType: 'url',
+        frequency: 6,
+        startTime: new Date('August 25, 2017').getTime(),
+        endTime: new Date('October 1, 2017 23:59:50').getTime(),
+        url: 'https://signup.take-part-in-research.service.gov.uk/?utm_campaign=Scrutinise_government&utm_source=Hold_gov_to_account&utm_medium=gov.uk&t=GDS&id=15',
+        templateArgs: {
+          title: 'Help improve GOV.UK',
+          surveyCta: 'Answer some questions about yourself to join the research community.',
+          surveyCtaPostscript: 'This link opens in a new tab.'
+        },
+        activeWhen: {
+          path: [
+            '^/government/policies(?:/|$)',
+            '^/government/announcements(?:/|$)',
+            '^/government/publications(?:/|$)',
+            '^/government/statistics(?:/|$)',
+            '^/government/publications?publication_filter_options=consultations(?:/|$)',
+            '^/government/organisations/hm-revenue-customs(?:/|$)',
+            '^/government/organisations/companies-house(?:/|$)',
+            '^/government/organisations/driver-and-vehicle-licensing-agency(?:/|$)',
+            '^/government/organisations/uk-visas-and-immigration(?:/|$)',
+            '^/government/organisations/land-registry(?:/|$)'
```

Comment:
As many of these pages are index pages that have lots of subpages (for example /government/publications/ is the root for almost every document published via whitehall, we need to check if the intention is that we only show the survey on the index, or if we show it on all child pages too.  I suspect in this case we only mean the index in which case we have to change the regexs here.  We've got `(?:/|$)` (this matches a `/` or the end of the string) as a suffix which will match `/government/publications`, `/government/publications/` and `/government/publications/anything` (but not `/government/publications-about-something-else`).  To avoid matching the third form we can change the suffix to: `/?$` (this matches the end of the string or a `/` followed by the end of the string).  I've asked Sean to confirm.

Reply:
We've decided that we should treat all these urls as exact matches rather than prefix matches, so the survey only appears on these exact pages and not any of their children pages.  The urls for the other surveys are all ok as they're already specific pages, or (as already mentioned for `becoming_a_childminder` are prefix paths that we want all the child pages of anyway).